Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.py

```diff
@@ -0,0 +1,395 @@
+                                  outputs=[],
+                                  readable_output=readable_message,
+                                  raw_response=[])
+        else:
+            parsed_url = urlparse(next_link)
+            skip_token = parse_qs(parsed_url.query)['$skiptoken'][0]
+
+    raw_response = client.risky_users_list(risk_state,
+                                           risk_level,
+                                           limit,
+                                           skip_token)
+
+    table_headers = ['id', 'userDisplayName', 'userPrincipalName', 'riskLevel',
+                     'riskState', 'riskDetail', 'riskLastUpdatedDateTime']
+
+    outputs = raw_response.get('value', {})
+
+    table_outputs = [create_event_or_incident_output(item, table_headers)
+                     for item in outputs]
+
+    readable_output = tableToMarkdown(name=f'Risky Users List\n'
+                                           f'Current page size: {args["limit"]}\n'
+                                           f'Showing page {args["page"]} out others that may exist',
+                                      t=table_outputs,
+                                      headers=table_headers,
+                                      removeNull=True,
+                                      headerTransform=pascalToSpace)
+
+    return CommandResults(outputs_prefix='AzureRiskyUsers.RiskyUser',
+                          outputs_key_field='id',
+                          outputs=outputs,
+                          readable_output=readable_output,
+                          raw_response=raw_response)
+
+
+def risky_user_get_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Get a risky user by ID.
+
+    Args:
+        client (Client): Azure Risky Users API client.
+        args (dict): Arguments for API call.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    raw_response = client.risky_user_get(args['id'])
+
+    table_headers = ['id', 'userDisplayName', 'userPrincipalName', 'riskLevel',
+                     'riskState', 'riskDetail', 'riskLastUpdatedDateTime']
+
+    outputs = create_event_or_incident_output(raw_response, table_headers)
+
+    readable_output = tableToMarkdown(name=f'Found Risky User With ID: {raw_response.get("id")}',
+                                      t=outputs,
+                                      headers=table_headers,
+                                      removeNull=True,
+                                      headerTransform=pascalToSpace)
+
+    return CommandResults(outputs_prefix='AzureRiskyUsers.RiskyUser',
+                          outputs_key_field='id',
+                          outputs=raw_response,
+                          readable_output=readable_output,
+                          raw_response=raw_response)
+
+
+def risk_detections_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
```

Comment:
this function's code is mostly a duplicate of `risky_users_list_command`.
Let's avoid that by extracting common functionalities to helper functions.
It will also make the code more readable